src/sentry/scim/endpoints/members.py

```diff
@@ -399,7 +399,7 @@ def get(self, request: Request, organization) -> Response:
         ).order_by("email", "id")
         if query_params["filter"]:
             filtered_users = user_service.get_many_by_email(
-                emails=query_params["filter"],
+                emails=[query_params["filter"]],
```

Comment:
This was the main issue

Reply:
can we add a test where there are multiple emails in this array of emails?

Reply:
It only accepts 1 name